src/test/java/org/apache/commons/imaging/formats/bmp/BmpImageParserTest.java

```diff
@@ -16,6 +16,7 @@
  */
 package org.apache.commons.imaging.formats.bmp;
 
+import static org.apache.commons.imaging.test.TestResources.fileResource;
```

Comment:
I think we should only use static imports for JUnit.

Reply:
Have changed it to regular imports